lib/core/viewer/components/viewer.component.spec.ts

```diff
@@ -962,25 +965,27 @@ describe('ViewerComponent', () => {
             });
 
             it('should update version when emitted by image-viewer and user has update permissions', () => {
+                spyOn(uploadService, 'uploadFilesInTheQueue').and.callFake(() => {});
                 component.readOnly = false;
                 component.nodeEntry = new NodeEntry({ entry: { name: 'fakeImage.png', id: '12', content: { mimeType: 'txt' } } });
                 const data = atob('iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVR42mNk+M9QDwADhgGAWjR9awAAAABJRU5ErkJggg==');
                 const fakeBlob = new Blob([data], { type: 'image/png' });
                 component.onSubmitFile(fakeBlob);
                 fixture.detectChanges();
 
-                expect(component.blobFile).toEqual(fakeBlob);
+                expect(uploadService.uploadFilesInTheQueue).toHaveBeenCalled();
```

Comment:
why are we changing that? before we were checking the file and now only that a method is called which does not make any sense.

Reply:
because that line with blob was added by mistake, we don't need that

Reply:
i'll also add an expect to check if the upload was called with the right BLOB.